site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -0,0 +1,165 @@
+    Name:                  sample-admissioncheck
+    Pod Set Updates:
+      Annotations:
+        cluster-autoscaler.kubernetes.io/consume-provisioning-request:  sample-job-2zcsb-57864-sample-admissioncheck-1
+        cluster-autoscaler.kubernetes.io/provisioning-class-name:       queued-provisioning.gke.io
+      Name:                                                             main
+    State:                                                              Ready
+```
+
+## What is the current state of my Provisioning Request?
+
+One of the reason your job is not running is that ProvisioningRequest is waiting to be provisioned. To find out if this is the case you can view Provisioning Request's state by running the following command:
+
+```bash
+kubectl get provisioningrequest PROVISIONING_REQUEST_NAME
+```
+
+If this is the case, the output should look similar to this:
+
+```bash
+NAME                                                 ACCEPTED   PROVISIONED   FAILED   AGE
+sample-job-2zcsb-57864-sample-admissioncheck-1       True       False         False    20s
+```
+
+You can also view more detailed status of your ProvisioningRequest by running the following command:
+
+```bash
+kubectl describe provisioningrequest PROVISIONING_REQUEST_NAME
+```
+
+If you ProvisioningRequest fails to provision nodes, the error output may look similar to this:
+```bash
+[...]
+Status:
+  Conditions:
+    Last Transition Time:  2024-05-22T13:04:54Z
+    Message:               Provisioning Request wasn't accepted.
+    Observed Generation:   1
+    Reason:                NotAccepted
+    Status:                False
+    Type:                  Accepted
+    Last Transition Time:  2024-05-22T13:04:54Z
+    Message:               Provisioning Request wasn't provisioned.
+    Observed Generation:   1
+    Reason:                NotProvisioned
+    Status:                False
+    Type:                  Provisioned
+    Last Transition Time:  2024-05-22T13:06:49Z
+    Message:               max cluster limit reached, nodepools out of resources: default-nodepool (cpu, memory)
+    Observed Generation:   1
+    Reason:                OutOfResources
+    Status:                True
+    Type:                  Failed
+```
+
+Note that the `Reason` and `Message` values for `Failed` condition may differ from your output, depending on an error.
+
+Provisioning Request state is described in the `.conditions[*].status` field.  Empty field means ProvisinongRequest is still being processed by the Cluster Autoscaler. Otherwise it falls into one the states listed below:
+- `Accepted` - indicates that the ProvisioningRequest was accepted by ClusterAutoscaler, so ClusterAutoscaler will attempt to provision the nodes for it.
+- `Provisioned` - indicates that all of the requested resources were created and are available in the cluster. Cluster Autoscaler will set this condition when the VM creation finishes successfully.
```

Comment:
```suggestion
- `Provisioned` - indicates that all of the requested resources were created and are available in the cluster. ClusterAutoscaler will set this condition when the VM creation finishes successfully.
```

Let's use the technical term consistently. "ClusterAutoScaler" vs "Cluster Autoscaler"